synapse/lib/drive.py

```diff
@@ -69,6 +69,9 @@ def _reqInfoType(self, info, typename):
 
     def getItemInfo(self, iden, typename=None):
         info = self._getItemInfo(s_common.uhex(iden))
+        if not info:
```

Comment:
if you supply a typename and the drive doesn't have that iden, you get an unhandled python Attribute error about NoneType not having a `get` attribute.